extensions/windows-event-log/tests/CWELTestUtils.h

```diff
@@ -94,3 +94,13 @@ class OutputFormatTestController : public TestController {
   std::string output_format_;
   std::optional<std::string> json_format_;
 };
+
+void generateLogFile(const std::wstring& channel, const std::filesystem::path& path) {
+  HANDLE event_log = OpenEventLog(NULL, std::string(channel.begin(), channel.end()).c_str());
```

Comment:
We should use RAII to close the handle. (`unique_ptr` with deleter, or `gsl::finally`)
Now it's leaked when we throw.

Reply:
done